01week/datatypes.js

```diff
@@ -1,24 +1,22 @@
 //Display current Date and Time
-var usaTime = new Date().toLocaleString("en-US", {
-  timeZone: "America/Chicago"
```

Comment:
Wow- Chicago Specific, are you from that area?

Reply:
most apps use Chicago for central time